packages/build/src/plugins_core/functions/index.ts

```diff
@@ -19,6 +20,45 @@ const getBundlers = (results: Awaited<ReturnType<typeof zipFunctions>> = []) =>
       .filter(Boolean) as NodeBundlerName[],
   )
 
+// see https://docs.netlify.com/functions/trigger-on-events/#available-triggers
+const eventTriggeredFunctions = new Set([
+  'deploy-building',
+  'deploy-succeeded',
+  'deploy-failed',
+  'deploy-deleted',
+  'deploy-locked',
+  'deploy-unlocked',
+  'submission-created',
+  'split-test-activated',
+  'split-test-deactivated',
+  'split-test-modified',
+  'identity-validate',
+  'identity-signup',
+  'identity-login',
+])
+
+const validateCustomRoutes = function (zisiResults: Awaited<ReturnType<typeof zipFunctions>>) {
+  for (const { routes, name, schedule } of zisiResults) {
+    if (!routes || routes.length === 0) continue
+
+    if (schedule) {
+      const error = new Error(
+        `Scheduled functions must not specify a custom path. Please remove the "path" configuration or pick a different name for the function. Learn more about scheduled functions at https://ntl.fyi/custom-path-scheduled-functions.`,
```

Comment:
Hmm how will picking a different name for the function help in the case of a scheduled function?

Reply:
copy & paste error!

Reply:
fixed in [4a5a13b](https://github.com/netlify/build/pull/5344/commits/4a5a13bfd3604de39799825ec3dc8146092eb0ad)